app/src/main/java/org/simple/clinic/onboarding/OnboardingScreen.kt

```diff
@@ -48,6 +51,9 @@ class OnboardingScreen(context: Context, attributeSet: AttributeSet) : Constrain
     setIntroOneTextView()
     setIntroTwoTextView()
     setIntroThreeTextView()
+    getStartedButton.setOnClickListener {
+      findNavController().navigate(R.id.action_onboardingScreen_to_selectCountryScreen)
+    }
```

Comment:
We should make this part of the Mobius loop, otherwise the button click events will not be forwarded to analytics.